docs/guide/supported-computing-services.md

```diff
@@ -324,15 +324,28 @@ Scheduling tasks on [Compute Engine](#google-compute-engine) has one big disadva
 start which usually takes around a minute. One minute is not that long but can't compete with hundreds of milliseconds
 that takes a container cluster on GKE to start a container.
 
-To start scheduling tasks on a container cluster we first need to create one using `gcloud`. Here is a command to create
-an auto-scalable cluster that will scale down to zero nodes when there is no load for some time and quickly scale up with
-the load during peak hours:
+To start scheduling tasks on a container cluster we first need to create one using `gcloud`. Here is a recommended configuration
+of a cluster that is very similar to what is used for the managed `contianer` instances. We recommend creating a cluster with two node pools:
+
+* `default-pool` with a single node and no autoscaling for system pods required by Kubernetes.
+* `workers-pool` that will use [Compute-Optimized instances](https://cloud.google.com/compute/docs/machine-types#compute-optimized_machine_type_family)
+  and SSD storage for better performance. This pool also will be able to scale to 0 when there are no tasks to run.
 
 ```yaml
 gcloud container clusters create cirrus-ci-cluster \
+  --autoscaling-profile optimize-utilization \
```

Comment:
I don't quite understand the reason behind adding an autoscaling profile to the pool with no autoscaler. Is it a workaround to achieve something? Also, how about adding `--node-taints dedicated=system:PreferNoSchedule` to this pool?

Reply:
Profile is for the cluster itself so it will try to utilize each node in any pool as much as possible instead of evenly distributing the load between nodes. This way is will much more cost efficient.

Not sure about the taints if they'll do anything. I think scheduler should be able to reschedule system pods 🤔  But since it's `PreferNoSchedule` I don't see why not to add it to ease life of the scheduler. Added in 8850f44